clikit/src/main/java/org/adamalang/clikit/model/ArgDefinition.java

```diff
@@ -0,0 +1,39 @@
+package org.adamalang.clikit.model;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+
+import java.util.HashMap;
+
```

Comment:
Let's document this and how this relates to an Argument, and let's refactor this name from ArgDefinition to ArgumentDefintion to make the relationship clear